packages/react-renderer-demo/src/app/src/doc-components/custom-buttons.js

```diff
@@ -36,22 +36,32 @@ const FormTemplate = ({ formFields, schema }) => {
             <Button disabled={submitting || !valid} style={{ marginRight: 8 }} type="submit" color="primary" variant="contained">
               Submit
             </Button>
-            <Button disabled={pristine} style={{ marginRight: 8 }} onClick={reset} variant="contained">
+            <Button disabled={pristine} style={{ marginRight: 8 }} onClick={() => { onReset(); reset(); }} variant="contained">
```

Comment:
we have `canReset` prop to show/hide the reset button. Cancel should be also wrapper in condition

Reply:
I would rather keep this as a minimalist example. Our formTemplate is more complex, there are more props (buttonOrders, etc.), so the example could become too big (and I don't think that we should show users how to use conditions in JSX :D )... and anyone can look on the code of our templates.